code/modules/research/research.dm

```diff
@@ -125,18 +118,17 @@ research holder datum.
 	var/datum/tech/KT = known_tech[ID]
 	if(KT)
 		if(KT.level <= level)
-			KT.level = max((KT.level + 1), (level - 1))
-
-/datum/research/proc/UpdateDesigns(obj/item/I, list/temp_tech)
-	for(var/v in known_designs)
-		var/datum/design/D = known_designs[v]
-		var/shared_tech = length(temp_tech & D.req_tech)
-		if(shared_tech)
-			D.reliability = min(100, D.reliability + shared_tech)
-			if(D.build_path == I.type)
-				D.reliability = min(100, D.reliability + shared_tech * rand(1,3))
-				if(I.crit_fail)
-					D.reliability = min(100, D.reliability + shared_tech * rand(3, 5))
+			KT.level = KT.level + 1
+
+//Checks if the origin level can raise current tech levels
+//Input: Tech's ID and Level; Output: TRUE for yes, FALSE for no
+/datum/research/proc/IsTechHigher(ID, level)
+	var/datum/tech/KT = known_tech[ID]
+	if(KT)
+		if(KT.level <= level)
+			return TRUE
+		else
+			return FALSE

```

Comment:
In this file: removal of reliability check procs and adds a proc to simply check if the tech level can raise existing tech levels